docs/using-controller-manager-with-kubeadm.md

```diff
@@ -27,42 +18,15 @@
     kubeadm init --config kubeadm.conf
     ```
 
-    You can find an example kubeadm.conf in `manifests/controller-manager/kubeadm.conf`. Follow the usual steps to install the network plugin and then bootstrap the other nodes using `kubeadm join`.
-     >Note, your nodes may not have an External IP address. This will cause logs and CNI to have have issues until the cluster provider is complete. Reference https://github.com/kubernetes/kubernetes/pull/75229 for further information.
-
-- Allow controller-manager to have access `/etc/kubernetes/cloud-config`.
-
-    Modify `/etc/kubernetes/manifests/kube-controller-manager.yaml` file to mount an extra volume to the controller manager pod.
-
-    ```
-    ...
-        volumeMounts:
-        - mountPath: /etc/kubernetes/cloud-config
-          name: cloud-config
-          readOnly: true
-    ...
-      volumes:
-      - name: cloud-config
-        hostPath:
-          path: /etc/kubernetes/cloud-config
-          type: FileOrCreate
-    ...
-    ```
-
-    Then wait for the controller manager to be restarted and running.
+- Boostrap any additional master nodes and worker nodes. You will need to add `KUBELET_EXTRA_ARGS="--cloud-provider=external"` to `/etc/default/kubelet` before running `kubeadm join`. 
```

Comment:
this seems to be platform specific. https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/kubelet-integration/#the-kubelet-drop-in-file-for-systemd
Please add OS on which you have tested this install guide as note, at the top of the guide.

Reply:
Ah sorry. Yeah, per the doc:

> KUBELET_EXTRA_ARGS is sourced from /etc/default/kubelet (for DEBs), or /etc/sysconfig/kubelet (for RPMs)

I will note the OS I used and add this note as well.